test/compilable/extra-files/ddoc9764.html

```diff
@@ -464,8 +464,8 @@
     <div class="content_wrapper">
       <article class="module">
         <h1 class="module_name">ddoc9764</h1>
-        <section id="module_content">
-
+        <section id="module_content"><br><br>
+<br><br>

```

Comment:
@CyberShadow how do you mean that? Remove the line consisting of `<br><br>`? I have a much more evil plan about that :)